CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
+# 1.10.0
```

Comment:
I think that the CHANGELOG.md need to follow this format:

https://keepachangelog.com/en/1.0.0/

ref: https://docs.nextcloud.com/server/latest/developer_manual/app_publishing_maintenance/release_process.html?highlight=changelog#change-log

Reply:
Oh, good catch! Updated now.